data/etc/apache2/includes/auth-pubtkt.conf

```diff
@@ -0,0 +1,8 @@
+TKTAuthPublicKey /usr/share/sso-keys/pubkey.pem
+TKTAuthLoginURL /plinth/accounts/sso/login/
+TKTAuthBackArgName next
+TKTAuthDigest SHA1
+TKTAuthRefreshURL /plinth/accounts/sso/refresh/
+AuthType mod_auth_pubtkt
+AuthName "FreedomBox Single Sign On"
+Require valid-user
```

Comment:
I wonder if we should change the name of this file, to indicate that it belongs to plinth / freedombox.